Subject: Various fixes for libpcap0.8 compilation.
Author: Romain Francoise <rfrancoise@debian.org>
Last-Update: 2014-09-02

Index: hping3.git/libpcap_stuff.c
===================================================================
--- hping3.git.orig/libpcap_stuff.c
+++ hping3.git/libpcap_stuff.c
@@ -17,7 +17,6 @@
 #include <stdlib.h>
 #include <sys/ioctl.h>
 #include <pcap.h>
-#include <net/bpf.h>
 
 #include "globals.h"
 
Index: hping3.git/script.c
===================================================================
--- hping3.git.orig/script.c
+++ hping3.git/script.c
@@ -24,7 +24,6 @@
 
 #include <sys/ioctl.h>
 #include <pcap.h>
-#include <net/bpf.h>
 
 #include "release.h"
 #include "hping2.h"
Index: hping3.git/configure
===================================================================
--- hping3.git.orig/configure
+++ hping3.git/configure
@@ -106,7 +106,7 @@ fi
 # configurable stuff
 #
 PCAP="PCAP=-lpcap"
-PCAP_INCLUDE=""
+PCAP_INCLUDE="-I/usr/include/pcap"
 
 for ARG in $*; do
 	case "$ARG" in
