Subject: Enable CFLAGS, CPPFLAGS and LDFLAGS in upstream MakeFile
Author: Guillaume Delacour <gui@iroqwa.org>
Last-Update: 2014-09-02

Index: hping3.git/Makefile.in
===================================================================
--- hping3.git.orig/Makefile.in
+++ hping3.git/Makefile.in
@@ -50,7 +50,7 @@ libars.a: $(ARSOBJ)
 	$(RANLIB) $@
 
 hping3: byteorder.h $(OBJ)
-	$(CC) -o hping3 $(CCOPT) $(DEBUG) $(OBJ) -L$(prefix)/lib $(PCAP) @SOLARISLIB@ @TCL_LIB@
+	$(CC) -o hping3 $(CCOPT) $(CPPFLAGS) $(DEBUG) $(OBJ) -L$(prefix)/lib $(PCAP) @SOLARISLIB@ @TCL_LIB@ $(LDFLAGS)
 	@echo
 	./hping3 -v
 	@echo "use \`make strip' to strip hping3 binary"
@@ -63,7 +63,7 @@ byteorder.h:
 	./configure
 
 .c.o:
-	$(CC) -c $(CCOPT) $(DEBUG) $(COMPILE_TIME) $<
+	$(CC) -c $(CFLAGS) $(CPPFLAGS) $(CCOPT) $(DEBUG) $(COMPILE_TIME) $<
 
 clean:
 	rm -rf hping3 *.o libars.a
